### DISCUSSION TITLE
Sample demo in ios crash

### DISCUSSION THREAD

**Discussion by neciszhang** [pmndrs/react-three-rapier#424] [Q&A]
I use a sample demo in ios. It crash.
`<Physics debug>
          <RigidBody
            colliders={'ball'}
            restitution={2}
          >
            <Torus />
          </RigidBody>
          <CuboidCollider position={[0, -2, 0]} args={[20, .5, 20]} />
        </Physics>`

**Comment by wiledal** [comment]
Hello!

Can you be more specific?

- Do you have a link to the page you were viewing
- What version of iOS are you using? 
- What device (example: iPhone 12)

**Reply by neciszhang** [reply]
> Hello!
> 
> Can you be more specific?
> 
> * Do you have a link to the page you were viewing
> * What version of iOS are you using?
> * What device (example: iPhone 12)

When I open 'https://react-three-rapier.pmnd.rs/'
ios: 14.3
device: iPhone 12
safari browser

**Comment by neciszhang** [comment]
> Hello!
> 
> Can you be more specific?
> 
> * Do you have a link to the page you were viewing
> * What version of iOS are you using?
> * What device (example: iPhone 12)
Hi, [here is crash problem](https://github.com/dimforge/rapier.js/issues/14)
